Formula/code-server.rb

```diff
@@ -31,13 +31,8 @@ def install
     system "yarn", "--production", "--frozen-lockfile"
     # @parcel/watcher bundles all binaries for other platforms & architectures
     # This deletes the non-matching architecture otherwise brew audit will complain.
-    arch = `uname -m`
-    path_to_prebuilds = "vendor/modules/code-oss-dev/node_modules/@parcel/watcher/prebuilds"
-    if arch == "arm64"
-      remove_dir("#{path_to_prebuilds}/darwin-x64")
-    else
-      remove_dir("#{path_to_prebuilds}/darwin-arm64")
-    end
+    prebuilds = testpath/"vendor/modules/code-oss-dev/node_modules/@parcel/watcher/prebuilds"
```

Comment:
```suggestion
    prebuilds = buildpath/"vendor/modules/code-oss-dev/node_modules/@parcel/watcher/prebuilds"
```
Oops, sorry, my mistake